examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -3008,8 +3076,10 @@
       },
       "required": [
         "uuid",
-        "int64_value",
-        "double_value"
+        "int64Value",
+        "doubleValue",
+        "floatValue",
```

Comment:
Note that the casing has been fixed for `int64_value` and `double_value`. `float_value` is also added to the array now where previously it was not.